app/config/config.yml

```diff
@@ -38,7 +38,6 @@ framework:
         engines: ['twig']
     default_locale:  "%locale%"
     trusted_hosts:   ~
-    trusted_proxies: "%trusted_proxies%"
```

Comment:
is this setting deprecated? if so you should also remove param from parameters.yml.dist file